device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttConnection.java

```diff
@@ -1,245 +0,0 @@
-/*
- *  Copyright (c) Microsoft. All rights reserved.
- *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
- */
-
-package com.microsoft.azure.sdk.iot.device.transport.mqtt;
```

Comment:
This class had very little meaningful code in it, since it just delegated every API to the mqttAsyncClient. I moved the important code out of this class and then deleted it